src/Sparrow/LowMemory/MemoryInformation.cs

```diff
@@ -77,73 +78,98 @@ public struct MemoryStatusEx
             {
                 if (Logger.IsInfoEnabled)
-                    Logger.Info($"Failed to read {filter} from {path}. Line was {parsedLine}");
-                return 0;
+                    Logger.Info($"Failed to read " + string.Join(",", filters) + " from {path}");
+                return null;
             }
 
-            if (parsedLine[0].Contains($"{filter}:") == false)
+            var cnt = 0;
+            foreach (var filterString in filtersString)
             {
-                if (Logger.IsInfoEnabled)
-                    Logger.Info($"Failed to find {filter} from {path}. Line was {parsedLine}");
-                return 0;
-            }
+                var parsedLine = filterString.Split(new[] {' '}, StringSplitOptions.RemoveEmptyEntries);
 
-            if (long.TryParse(parsedLine[1], out var result) == false)
-            {
-                if (Logger.IsInfoEnabled)
-                    Logger.Info($"Failed to parse {filter} from {path}. Line was {parsedLine}");
-                return 0;
-            }
+                if (parsedLine.Length != 3) // format should be: {filter}: <num> kb
+                {
+                    if (Logger.IsInfoEnabled)
+                        Logger.Info($"Failed to read {filters[cnt]} from {path}. Line was {parsedLine}");
+                    rcValues[cnt++] = 0;
+                    continue;
+                }
 
-            switch (parsedLine[2].ToLowerInvariant())
-            {
-                case "kb":
-                    result *= 1024L;
-                    break;
-                case "mb":
-                    result *= 1024L * 1024;
-                    break;
-                case "gb":
-                    result *= 1024L * 1024 * 1024;
-                    break;
+                var found = false;
+                foreach (var filter in filters)
+                {
+                    if (parsedLine[0].Contains($"{filter}:") == true)
+                    {
+                        found = true;
+                        break;
+                    }
+                }
+
+                if (found == false)
+                    continue;
+
+                if (long.TryParse(parsedLine[1], out var result) == false)
+                {
+                    if (Logger.IsInfoEnabled)
+                        Logger.Info($"Failed to parse {filters[cnt]} from {path}. Line was {parsedLine}");
+                    rcValues[cnt++] = 0;
+                    continue;
+                }
+
+                switch (parsedLine[2].ToLowerInvariant())
+                {
+                    case "kb":
+                        result *= 1024L;
+                        break;
+                    case "mb":
+                        result *= 1024L * 1024;
+                        break;
+                    case "gb":
+                        result *= 1024L * 1024 * 1024;
+                        break;
+                }
+
+                rcValues[cnt++] = result;
```

Comment:
Isn't the parsing code the same as in the debug endpoint handler? Can we extract it and have it in one place?